skyvern/webeye/actions/handler.py

```diff
@@ -362,12 +374,13 @@ async def handle_select_option_action(
                 "SelectOptionAction on a non-listbox element. Cannot handle this action",
             )
             return [ActionFailure(Exception(f"Cannot handle SelectOptionAction on a non-listbox element"))]
-    elif tag_name == "input" and await locator.get_attribute(
-        "type", timeout=SettingsManager.get_settings().BROWSER_ACTION_TIMEOUT_MS
-    ) in ["radio", "checkbox"]:
-        # TODO: double click will uncheck the checkbox
-        click_action = ClickAction(element_id=action.element_id)
-        return await chain_click(task, page, click_action, xpath)
+    elif tag_name == "input" and element_dict.get("attributes", {}).get("type", None) in ["radio", "checkbox"]:
+        LOG.info(
+            "SelectOptionAction is on <input> checkbox/radio",
+            action=action,
+        )
+        check_action = CheckboxAction(element_id=action.element_id, is_checked=True)
```

Comment:
With this change we're losing the ability to uncheck checkboxes, right?

Reply:
Yes, you're right.
But I think it's ok since we don't expect LLM to return some actions to do the uncheck.
I think we only assume LLM will return actions to check.

Reply:
For the most usecases today that's correct, it's ok for this PR. Can you create a ticket for it so we won't forget?